src/components/ViewPCOL/Details.tsx

```diff
@@ -0,0 +1,122 @@
+import { Label, Text, Title2 } from "@fluentui/react-components";
+import { usePCOL } from "src/api/PCOL/usePCOL";
+import { useParams } from "react-router-dom";
+
+const ViewPCOLDetails = () => {
+  const { program, pcolId } = useParams();
+  const pcol = usePCOL(String(program), Number(pcolId));
+  console.log(pcol.data?.Disclaimers);
```

Comment:
You probably don't need this console.log anymore -- look at others and see if they can be removed too